sourcemash/frontend/assets.py

```diff
@@ -39,8 +39,10 @@
                 "js/views/feed_view.js",
                 "js/views/saved_view.js",
                 "js/views/browse_view.js",
+                "js/views/register_view.js",
```

Comment:
It's not hard to do if you think it's useful, but isn't that design a bit odd? 

Maybe we should add a modal footer that says "Already have an account", and when you click on it, it activates the login form in the sidenav (not sure how that would work with the Backbone views). Would you like to raise an issue for that?